themes/src/main/resources/theme/base/admin/resources/partials/realm-tokens.html

```diff
@@ -3,6 +3,19 @@
 
     <form class="form-horizontal" name="realmForm" novalidate kc-read-only="!access.manageRealm">
 
+        <div class="form-group">
+            <label class="col-md-2 control-label" for="defaultSignatureAlgorithm">{{:: 'default-signature-algorithm' | translate}}</label>
+
+            <div class="col-md-6">
+                <select id="defaultSignatureAlgorithm" class="form-control" ng-model="realm.defaultSignatureAlgorithm"
+                        ng-options="provider for provider in serverInfo.listProviderIds('signature')">
+                </select>
+            </div>
+
+            <kc-tooltip>{{:: 'default-signature-algorithm.tooltip' | translate}}
+            </kc-tooltip>
+        </div>
+
```

Comment:
I wonder whether we should extend "Tokens" page with similar settings for SAML client defaults (as another issue)

Reply:
Wouldn't it just use the realm default like OIDC?

Reply:
There are no realm defaults AFAIK.